scripts/release/mule/test/tests/post/verify_package_binaries.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+# shellcheck disable=2116
+
+set -e
+
+echo "[$0] Verifying installed binaries..."
+
+RET=0
+RPMTMP=$(mktemp -d)
+
+cd "./tmp/node_pkgs/$OS_TYPE/$ARCH_TYPE"
+
+if [ "$PKG_TYPE" = deb ]
+then
+    dpkg -L algorand > "$RPMTMP/algorand.install"
+    dpkg -L algorand-devtools > "$RPMTMP/algorand-devtools.install"
+else
+    rpm -ql algorand > "$RPMTMP/algorand.install"
+    rpm -ql algorand-devtools > "$RPMTMP/algorand-devtools.install"
+fi
+
+ALGORAND_BINS=(
+    /usr/bin/algocfg
```

Comment:
Can the bin dir be factored out?

Reply:
Yes, but why?  It's better if we match the full path as stated in the control files, in my opinion.

Reply:
Seems redundant to specify /usr/bin/ for each binary rather than just the binary name.

Reply:
What if there's a false positive for `/etc/algocfg`?  The binaries should be in a specific location prescribed by the control files, which is what we should match.

Reply:
Agree, just saying that the /usr/bin could be appended in the for loop rather than repeated for each binary.

Reply:
Can we just move this along?  I'd really like to get unblocked.